### ISSUE TITLE
Fix apt errors during ops up in platform test

### ISSUE THREAD

**Issue by nickthecook** [nickthecook/ops#45]
```shell
ops t
Running 'bin/run.sh "bin/ops test" $TEST_PLATFORMS ' from ops.yml in environment 'dev'...
bin/run.sh: Running 'bin/ops test' on platforms: ops-debian
bin/run.sh: Mounting '/Users/nickthecook/src/ops' into the container at '/ops'.
bin/run.sh: Starting existing container 'ops-debian_bin_ops_test'...
/entrypoint.sh: loading SSH agent...
Agent pid 8
/entrypoint.sh: running 'bundler install'...
/entrypoint.sh: running 'ops up'...
/usr/bin/apt-get
[Apt] curl                                         FAILED
Error meeting Apt dependency 'curl':

/usr/bin/apt-get
[Apt] sl                                           FAILED
Error meeting Apt dependency 'sl':

[Gem] bundler                                      OK
[Gem] rerun                                        OK
[Gem] ejson                                        OK
[Dir] runtime_data                                 OK
[Custom] bundle install --quiet                    OK
[Custom] echo this is stdout                       OK
/entrypoint.sh: Running command: bin/ops test
Running 'environment=test bundle exec rspec --exclude-pattern 'spec/e2e/**/*_spec.rb'' from ops.yml in environment 'dev'...
 164/164 |============================================ 100 =============================================>| Time: 00:00:00

Finished in 0.60002 seconds (files took 0.69255 seconds to load)
164 examples, 0 failures
```

**Comment by nickthecook** [comment]
The equivalent commands succeed when run from the terminal in the test container, so there is likely a problem in `Dependencies::Apt`.

**Comment by nickthecook** [comment]
This was a bug introduced when moving the logic to execute a command from `Dependency` into its own `Executor` class.

Fixed in 0.11.pre2.
